packages/server/src/alerts/alerts.service.ts

```diff
@@ -31,10 +31,6 @@ export class AlertsService {
           });
           const isQueueOpen = await queue?.checkIsOpen();
           if (question.closedAt || !isQueueOpen) {
-            if (!question.closedAt) {
```

Comment:
Briefly talked to Iris about this, but this seems kinda like what the clean queue service already does